quad_ufabc/Num thrust
quad_ufabc/Quaternion1 orientation_set_point